security/config.py

```diff
@@ -42,6 +42,8 @@
     'CELERY_STALE_TASK_TIME_LIMIT_MINUTES': 60,
     'BACKUP_STORAGE_CLASS': 'security.storages.BackupFileSystemStorage',
     'BACKUP_STORAGE_PATH': None,
+    'DEBUG_TOOLBAR': False,
+    'SHOW_DEBUG_TOOLBAR': False,
```

Comment:
Why have you decided to have two settings? Why not to have just one that enables toolbar logging and also displaying?

Reply:
because we can store data in the accept and then turned off its display later.